example-android/app/src/main/java/org/hiero/android_example/ui/main/CryptoTransferFragment.kt

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-package com.hedera.android_example.ui.main
```

Comment:
We should do this in multiple steps. This PR is too big to not oversee any issue ;)